scripts/skills/perks/perk_rf_the_rush_of_battle.nut

```diff
@@ -88,13 +86,12 @@ this.perk_rf_the_rush_of_battle <- ::inherit("scripts/skills/skill", {
 
 	function addStacks( _num )
 	{
-		local pool = this.m.CurrPool ? "StackPool1" : "StackPool2";
-		this.m[pool] = ::Math.min(this.m.MaxStacks, this.m[pool] + _num);
+		this.m.NewStacks = ::Math.min(this.m.MaxStacks, this.m.NewStacks + 1);
```

Comment:
`MaxStacks`s meaning is weird during `addStacks`: So with a MaxStacks of 5, you can have a total of 10 internal stacks, 5 can expire soon and 5 will expire a little bit later. 

I don't think there is a need to enforce MaxStacks during adding, when we enforce it during `getStacks`.

Reply:
It helps with the calculation in `getTooltip` where we show how many stacks will be lost at the end of the turn.